adafruit_led_animation/animation/sparkle.py

```diff
@@ -83,16 +87,20 @@ def _set_color(self, color):
         self._dim_color = dim_color
         self._sparkle_color = color
 
+    def _random_in_mask(self):
+        if len(self._mask) == 0:
```

Comment:
the one other pylint issue it has right now is that it wants this if statement to not use the `else:` like this:
```python
if len(self._mask) == 0:
    return random.randint(0, (len(self.pixel_object) - 1))
return self._mask[random.randint(0, (len(self._mask) - 1))]
```
Honestly I kind if prefer it written more explicitly like how you did it with the `else:` personally. But I'll leave it up to you, or we can see if anyone else has thoughts. You can either change it like above to make pylint happy, or add a disable comment in this in this function to have it ignore the check.

Reply:
no problem, i'll fix...